demo/monolayer_growth/src/cycling_cell.h

```diff
@@ -0,0 +1,86 @@
+// -----------------------------------------------------------------------------
+//
+// Copyright (C) 2021 CERN & University of Surrey for the benefit of the
+// BioDynaMo collaboration. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// See the LICENSE file distributed with this work for details.
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// -----------------------------------------------------------------------------
+
+#ifndef CYCLING_CELL_H_
+#define CYCLING_CELL_H_
+
+#include "biodynamo.h"
+#include "core/agent/cell.h"
+
+namespace bdm {
+
+class CyclingCell : public Cell {  // our object extends the Cell object
+                                   // create the header with our new data member
+  BDM_AGENT_HEADER(CyclingCell, Cell, 1);
+
+ public:
+  CyclingCell() {}
+  explicit CyclingCell(const Real3& position) : Base(position) {}
+  virtual ~CyclingCell() {}
+
+  /// If CyclingCell divides, the daughter has to initialize its attributes
+  void Initialize(const NewAgentEvent& event) override {
+    Base::Initialize(event);
+
+    if (auto* mother = dynamic_cast<CyclingCell*>(event.existing_agent)) {
+      cycle_ = mother->cycle_;
+      delt_ = mother->delt_;
+      vmax_ = mother->vmax_;
+      delt_ = mother->vinit_;
+      vrel_ = mother->vrel_;
+      if (event.GetUid() == CellDivisionEvent::kUid) {
+        // the daughter will be able to divide
+        can_divide_ = true;
+      } else {
+        can_divide_ = mother->can_divide_;
+      }
+    }
+  }
+
+  // getter and setter for division
+  void SetCanDivide(bool d) { can_divide_ = d; }
+  bool GetCanDivide() const { return can_divide_; }
+
+  // getter and setter for cells current phase of cell cycle
+  void SetCycle(int c) { cycle_ = c; }
+  int GetCycle() const { return cycle_; }
+
+  // getter and setter for cells maximum volume
+  void SetVmax(real_t vm) { vmax_ = vm; }
+  real_t GetVmax() const { return vmax_; }
+
+  // getter and setter for cells initial volume
+  void SetVinit(real_t vi) { vinit_ = vi; }
+  real_t GetVinit() const { return vinit_; }
+
+  // getter and setter for delt t
+  void SetDelt(real_t dt) { delt_ = dt; }
```

Comment:
`Delt` -> `Delta` for readability? Same for member, if we talk about a delta

Reply:
I think that is fine but in this case Delt t is specifically Delta t, so I reduced part of the delta. I suppose it could be specifically called Deltat ? Or just Delta for simplicity as you put.

Reply:
Renamed it to DeltaT.